src/collections/_documentation/workflow/alerts-notifications/alerts.md

```diff
@@ -0,0 +1,181 @@
+- Send a notification to a single legacy integration (service)
+    - "service" is the same as legacy integration
+- Send a notification to a global integration, which includes
+    - [PagerDuty](https://docs.sentry.io/workflow/integrations/global-integrations/#pagerduty)
+    - [Slack](https://docs.sentry.io/workflow/integrations/global-integrations/#slack)
+    - Custom integrations built using the [Integration Platform](https://docs.sentry.io/workflow/integrations/integration-platform/)
+
+[{% asset notifications/alert-actions2.png alt="The alert condition in this example shows the conditions are conflicting with each other." %}]({% asset notifications/alert-actions2.png @path %})
+
+For more details, see [LINK: Legacy Integrations (via “Service” as any individual integration) that can send Notifications].
+
+By default, [LINK: issue owners] receive alert emails. If an issue owner is not configured or not found, the email will either not send or send to all project members as defined in **Project Settings > Issue Owners**.
+
+[{% asset notifications/issue_owners.png alt="Issue owners can be toggled on or off." %}]({% asset notifications/issue_owners.png @path %})
+
+**Rate Limit**
+
+The rate limit determines how frequently an issue alerts. After an issue fires an alert, Sentry won't check the conditions and won't execute the actions for that issue until the rate limit period passes. The limit is set to perform the action according to one of these intervals:
+
+- minutes: 5, 10, 30, 60
+- hours: 3, 12, 24
+- one week or 30 days
+
+For example, if an issue violates alert conditions multiple times in a 30-min period, but your frequency threshold is 30 minutes, you’ll only get one alert.
+
+[{% asset notifications/alert_frequency.png alt="A dropdown that adjusts the frequency of an alert." %}]({% asset notifications/alert_frequency.png @path %})
+
+### Environment
+Environment control allows you to specify which environment qualifies for your issue alert.
+
+- All Environments
+- Any one of your defined (and not hidden) environments
+
+[{% asset notifications/alert_environment.png alt="A dropdown allowing you to choose an environment to attach to an alert." %}]({% asset notifications/alert_environment.png @path %})
+
+If you select "All Environments", Sentry checks individually for each environment, rather than combined events across environments.
+
+### Digests
+
+The digests feature works only for **issue alert emails** and limits alerts across projects. This project-level setting allows you to batch issue alerts to limit the total number of emails you receive for that project. Use the sliders to control the frequency.
+
+[{% asset notifications/alert_digest.png alt="A sliding adjustment scale for the frequency of alert emails." %}]({% asset notifications/alert_digest.png @path %})
+
+## Alert Listing
```

Comment:
Heads up on proposed changes here: https://www.notion.so/sentry/Rule-Page-Design-Doc-81108939d0f847938385942a16d1305d
Not likely to come soon though

Reply:
@adhiraj I'm assuming we can update these docs after Jeff's PR goes live?

Reply:
yep, we'll update it later